backend/tournesol/views/ratings.py

```diff
@@ -77,7 +77,7 @@ def get_serializer_class(self):
     def get_queryset(self):
         ratings = RatingsWithAnnotations.filter(
             user=self.request.user, n_comparisons__gt=0
-        )
+        ).select_related("video")
```

Comment:
Opening the "My rated videos" feels a lot slower for me than it should be (I have 401 videos when testing locally). I haven't looked enough at the backend code to figure the cause, nor do I know Django's internals enough. Could you quickly check if the endpoint performance seems right to you?

The query takes 800ms on my local environment:

![Screenshot 2021-12-19 at 09 36 44](https://user-images.githubusercontent.com/8818081/146668737-651a7b60-3be9-4412-9d68-b7f13489e171.png)

Reply:
Good catch. The `Count(filter=...)` query led to inefficient aggregations. I am now using an explicit `Subquery`. The code is a bit more verbose, but the SQL query is significantly simpler.

It should be fixed with 721991d